docs/faq.md

```diff
@@ -116,6 +117,13 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
 enables the playback of FLV video where the Flash plugin is available. For more information
 on media formats see the [troubleshooting guide][troubleshooting].
 
+## Q: How can I play RTMP video in video.js?
+
+Make sure that the Flash tech is available. Then, just set the rtmp source with
+an appropriate type like `rtmp/mp4`.
+The main thing to be aware of is that video.js splits the connection url and stream name with the `&` character.
+So, you'd want to update the url to follow that format. For example: `http://example.com/live&foo`.
```

Comment:
`rtmp://example.com/live&foo` or `rtmp://example.com/fms&mp4:path/to/file.mp4`.

If the server requires query parameters for authentication, these should be added to the connection part url, for example `rtmp://example.com/live?token=1234&foo`.